app/common/prescription/controllers/patientSimplifiedPrescriptionController.js

```diff
@@ -225,6 +230,17 @@
         });
     }
 
+    function isArvDrug(drug) {
```

Comment:
This function should be in [drugService](https://github.com/esaude/esaude-emr-poc/blob/master/app/common/domain/services/drugService.js). We should follow the [styleguide](https://github.com/johnpapa/angular-styleguide/tree/master/a1#data-services).